### ISSUE TITLE
Operations on Stack Data Structure

### ISSUE THREAD

**Issue by ItsAvi165** [PCCoE-Hacktoberfest-21/Data-Structures-and-Algorithms#51]

**Comment by ItsAvi165** [comment]
Can I work on this in C++ ?

**Comment by SumedhaZaware** [comment]
@ItsAvi165 yes, go for it!!

**Comment by ItsAvi165** [comment]
Ok thank you !

**Comment by OmkarBansode07** [comment]
Can you assign this issue to me?

**Comment by SumedhaZaware** [comment]
@OmkarBansode07 yup, if you complete the task using any language except CPP. Let me know if you are working

**Comment by OmkarBansode07** [comment]
@SumedhaZaware Yes, I can do this in another language.

**Comment by OmkarBansode07** [comment]
Thank you for assigning this issue

**Comment by Shishank93774** [comment]
I think I can work on it, mind me contributing?

**Comment by OmkarBansode07** [comment]
I can do in Python

**Comment by OmkarBansode07** [comment]
I am doing in Python

**Comment by SumedhaZaware** [comment]
@Shishank93774 You can work on using any language except python and cpp. Let me know if you are willing to work

**Comment by Shishank93774** [comment]
> @Shishank93774 You can work on using any language except python and cpp. Let me know if you are willing to work
Yeah, sure I would be glad to help with "java" as my language.
